igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/JenkinsBuildMonitor.groovy

```diff
@@ -108,21 +112,23 @@ class JenkinsBuildMonitor implements PollingMonitor {
 
     @Override
     void onApplicationEvent(ContextRefreshedEvent event) {
-        log.info('Started')
-        worker.schedulePeriodically(
-                {
-                    if (isInService()) {
-                        log.info "- Polling cycle started - ${new Date()}"
-                        buildMasters.filteredMap(BuildServiceProvider.JENKINS).keySet().parallelStream().forEach(
-                                { master -> changedBuilds(master) }
-                        )
-                        log.info "- Polling cycle done - ${new Date()}"
-                    } else {
-                        log.info("not in service (lastPoll: ${lastPoll ?: 'n/a'})")
-                        lastPoll = null
-                    }
-                } as Action0, 0, pollInterval, TimeUnit.SECONDS
-        )
+        if (jenkinsProperties.polling.enabled) {
```

Comment:
Would use a Spring @Value with a default of true instead of wiring this in jenkinsProperties and  bind that to the inService() method instead of current implementation